### PR TITLE
hip 118 visualization data 

### PR DESCRIPTION
**Pull request by andymck**
Adds usage count msgs to support hip 118 visualization data

### CODE REVIEW

File: src/service/poc_mobile.proto [hunk 1]
```diff
@@ -7,6 +7,70 @@ import "mapper.proto";
...
 
+message hex_usage_counts_req_v1 {
+  // H3 resolution 10 tile
+  uint64 hex = 1;
+  // Average count of unique helium mobile subscribers active in the hex during
+  // the previous 7 days ( inclusive of discovery mapping subscribers )
+  uint64 helium_mobile_subscriber_avg_count = 2;
+  // Average count of unique helium mobile discovery mapping subscribers active
+  // in the hex during the previous 7 days
+  uint64 helium_mobile_disco_mapping_avg_count = 3;
```
Comment (jeffgrunewald) [inline src/service/poc_mobile.proto:18]: probably shouldn't make these helium-mobile specific, right? `carrier_subscriber_avg_count` and such?

File: src/service/poc_mobile.proto [hunk 2]
```diff
@@ -7,6 +7,70 @@ import "mapper.proto";
...
+  // Average count of unique offload sessions active in the hex during the
+  // previous 7 days
+  uint64 offload_avg_count = 4;
+  // Average count of unique tmo cell tower sessions active in the hex during
+  // the previous 7 days
+  uint64 tmo_cell_avg_count = 5;
```
Comment (jeffgrunewald) [inline src/service/poc_mobile.proto:24]: assuming this is even a thing we're allowed to report (not restricted by any sort of service agreement) this should also probably be non-tmo specific; `mno_cell_avg_count` or similar

Comment (bbalser) [reply src/service/poc_mobile.proto:24]: agree with the naming. @madninja you verified that we are allowed to share this data?

Comment (bbalser) [reply src/service/poc_mobile.proto:24]: talked with ninja, lets rip out the tmo_cell_avg for right now

File: src/service/poc_mobile.proto (hunk 1 above)
Comment (andymck) [reply src/service/poc_mobile.proto]: renamed to service_provider_subscriber_avg_count and disco_mapping_avg_count

File: src/service/poc_mobile.proto (hunk 2 above)
Comment (andymck) [reply src/service/poc_mobile.proto]: removed

**Review by jeffgrunewald — APPROVED** [review]

**Review by bbalser — APPROVED** [review]

**Pull request merged**